types/filesize/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for filesize 3.2.1
+// Type definitions for filesize 3.6.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).